habitat-lab/habitat/config/benchmark/multi_agent/zxz_spot_drone_per.yaml

```diff
@@ -135,8 +119,8 @@ habitat:
       agent_0:
         joint_start_noise: 0.0
   dataset:
-    data_path: data/datasets/perception/perception_eval.json.gz
+    data_path: data/datasets/hssd_eval.json.gz
     # new
-    randomize_agent_start: 0
+    randomize_agent_start: 1
```

Comment:
Why would we  want to randomize agent initial position **BY DEFAULT**?

Reply:
Have solved it. It is now the same as before, and the robot configs has been uploaded on drive.